### PR TITLE
Refactored Feed Item to use Ember Test Selections

### PR DESCRIPTION
**Pull request by Alonski**
Hope I am doing this right...
@jonnii Please look over this and tell me if I am doing ember test selectors right...

### CODE REVIEW

**Comment by jonnii** [comment]
You need to do something like `this.$(testSelector('article-title', article.id)).text()` or something to that affect. The test selectors help you find elements on the page using data attributes.

**Comment by jonnii** [comment]
i usually pass in the same thing into the second arg of the helper, e.g. 

```
data-test-article-author-username=article.id
```

File: tests/integration/components/feed-item-test.js [hunk 1]
```diff
@@ -27,20 +29,40 @@ test('it renders', function(assert) {
...
-    this.$('img').attr('src'),
-    'https://static.productionready.io/images/smiley-cyrus.jpg',
+    testSelector('article-author-image', article.author.image),
+    '[data-test-article-author-image="https://static.productionready.io/images/smiley-cyrus.jpg"]',
     'Image is default image'
   );
 
-  assert.equal(this.$('a.author').text().trim(), 'Alon Bukai', 'Author name is correct');
+  assert.equal(
+    testSelector('article-author-username', article.author.username),
```
Comment (alexlafroscia) [inline tests/integration/components/feed-item-test.js]: I'm not sure this is testing what we actually want to test here. The idea isn't to verify the data based on the selectors, but to use the selectors to locate the DOM elements.  We'd want assert the text of the element or something like that, not that the attribute has the right value.

Comment (Alonski) [inline tests/integration/components/feed-item-test.js]: Yea 😅 I wasn't so sure how to use these.
Need to what @jonnii said:
`this.$(testSelector('article-title', article.id)).text()`

Comment (alexlafroscia) [inline tests/integration/components/feed-item-test.js]: 👍 bingo!

**Comment by alexlafroscia** [comment]
Nicely done @Alonski!

**Pull request merged**